src/main/scala/strawman/collection/Seq.scala

```diff
@@ -26,6 +31,9 @@ trait SeqOps[+A, +CC[X], +C] extends Any
   with ArrayLike[A]
   with Equals {
 
+  protected[this] def c: C
+  protected[this] def seq: Seq[A]
```

Comment:
I’m going to postpone these issues so that the 0.2.0 gets the contents of this PR. #134 keeps track of them.